### PR TITLE
Integrating Product Recs into Page Builder

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) adds content about how to add product recommendations to your Page Builder pages.

If Magento Commerce, please specify:

- [ ] Commerce only?
- [x] Commerce with B2B?

## Affected documentation pages

- https://docs.magento.com/m2/ee/user_guide/cms/page-builder-add-content.html
- https://docs.magento.com/m2/ee/user_guide/marketing/create-new-rec.html

### CODE REVIEW

**Comment by leslietilling** [comment]
@erikmarr  When is this due for release?

**Comment by erikmarr** [comment]
@leslietilling I believe we're shooting for Tuesday of next week.

**Comment by misha-kotov** [comment]
We pushed it back to Thu, 5/21

File: src/marketing/page-builder-add-product-recs.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+## Add an existing recommendation unit
+
+1. Navigate to the Page Builder workspace on the target page.
+
+1. In the Page Builder panel under **Add Content**, drag a **Product Recommendation** placeholder to the stage.
+
+    ![]({% link images/images-ee/page-builder-add-prex-drag.png %}){: .zoom}
+    _Select Product Recommendation_
+
+1. Hover over the empty product recommendation container. Then, choose **Settings** (![]({% link images/images-ee/icon-pb-settings.png %}){: .Inline}) in the toolbox and do the following:
```
Comment (leslietilling) [inline src/marketing/page-builder-add-product-recs.md:17]:
```suggestion
1. Hover over the empty product recommendation container, choose **Settings** (![]({% link images/images-ee/icon-pb-settings.png %}) in the toolbox, and do the following:
```

File: src/marketing/page-builder-add-product-recs.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+    -  Click <span class="btn">Select</span>.
+
+        ![]({% link images/images-ee/page-builder-add-prex-select.png %}){: .zoom}
+        _Select Product Recommendation_
+
+    -  In the list of active product recommendations, find the row with the recommendation unit that you want to add. In the last column, click <span class="btn">Select</span>.
```
Comment (leslietilling) [inline src/marketing/page-builder-add-product-recs.md:24]:
```suggestion
    -  In the list of active product recommendations, find the row with the recommendation unit that you want to add and click <span class="btn">Select</span> in the last column.
```

File: src/marketing/page-builder-add-product-recs.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+    -  Complete the remaining settings as needed, using the field descriptions at the end of this page for reference.
+
+    -  When complete, click <span class="btn">Save</span> to return to the Page Builder workspace.
+
+1. In the upper-right corner, click **Save**.
+
+## Edit recommendation unit settings
+
+1. Hover over the recommendation unit container. Then, choose **Settings** (![]({% link images/images-ee/icon-pb-settings.png %}){: .Inline}) in the toolbox and do the following:
```
Comment (leslietilling) [inline src/marketing/page-builder-add-product-recs.md:42]:
```suggestion
1. Hover over the recommendation unit container, choose **Settings** (![]({% link images/images-ee/icon-pb-settings.png %}) in the toolbox, and do the following:
```

File: src/marketing/page-builder-add-product-recs.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+    ![]({% link images/images-ee/page-builder-add-prex-toolbox.png %}){: .zoom}
+    _Block Toolbox_
+
+    -  To choose a different recommendation unit, click <span class="btn">Select Product Recommendation</span>. In the list of active recommendation units, <span class="btn">Select</span> the recommendation unit that you want to add. Then, click <span class="btn">Select</span>.
```
Comment (leslietilling) [inline src/marketing/page-builder-add-product-recs.md:47]: We should not use buttons as verbs in an instruction. You have a `Select` button twice here and I am not sure how this actually works to make a suggestion.

File: src/marketing/page-builder-add-product-recs.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+    -  Update the remaining settings as needed, using the field descriptions at the end of this page for reference.
+
+    -  When complete, click <span class="btn">Save</span> to return to the Page Builder workspace.
+
+1. In the upper-right corner, click <span class="btn">Save</span>.
+
+## Duplicate a recommendation unit
+
+1. Hover over the recommendation unit container. Then, choose **Duplicate** (![]({% link images/images-ee/icon-pb-duplicate.png %}){: .Inline}) in the toolbox.
```
Comment (leslietilling) [inline src/marketing/page-builder-add-product-recs.md:57]:
```suggestion
1. Hover over the recommendation unit container and choose **Duplicate** (![]({% link images/images-ee/icon-pb-duplicate.png %}) in the toolbox.
```

File: src/marketing/page-builder-add-product-recs.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+    The duplicate appears just below the original.
+
+1. To move the duplicated recommendation unit to a new position, hover over the container. Then, choose **Move** (![]({% link images/images-ee/icon-pb-move.png %}){: .Inline}) in the toolbox.
```
Comment (leslietilling) [inline src/marketing/page-builder-add-product-recs.md:61]:
```suggestion
1. To move the duplicated recommendation unit to a new position, hover over the container and choose **Move** (![]({% link images/images-ee/icon-pb-move.png %}) in the toolbox.
```

File: src/marketing/page-builder-add-product-recs.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+1. Select and drag the recommendation unit until the red guideline appears at the new position.
+
+    The top and bottom borders of each container appear as dashed lines while the recommendation unit is moved.
+
+## Remove a recommendation unit from the stage
+
+1. Hover over the recommendation unit container. Then, choose **Remove** (![]({% link images/images-ee/icon-pb-remove.png %}){: .Inline}) in the recommendation unit toolbox.
```
Comment (leslietilling) [inline src/marketing/page-builder-add-product-recs.md:69]:
```suggestion
1. Hover over the recommendation unit container and choose **Remove** (![]({% link images/images-ee/icon-pb-remove.png %}) in the recommendation unit toolbox.
```

File: src/marketing/page-builder-add-product-recs.md (hunk 4 above)
Comment (erikmarr) [reply src/marketing/page-builder-add-product-recs.md]: fixed

**Comment by erikmarr** [comment]
running tests

**Pull request merged**